scripts/import/import_variant_synonyms

```diff
@@ -433,6 +485,44 @@ sub extract_PharmGKB {
   return \%synonyms;
 }
 
+=head2 parse_eva_file
+
+parse EVA data file
+
+=cut
+
+sub parse_eva_file {
+  my $infile = shift;
+  
+  my %synonym;
+
+  # Open the input file for reading
+  my $fh;
+  if($infile =~ /gz$/) {
+    open ($fh, "zcat $infile |") or die ("Could not open $infile for reading\n");
+  }
+  else {
+    open ($fh, '<', $infile) or die ("Could not open $infile for reading\n");
+  }
+
+  # Read through the file and parse out the desired fields: 
+  # current rs id and synonym
+  while (my $row = <$fh>) {
+    chomp $row;
+    next if($row =~ /^#/);
+    
+    my @row_data = split /\t/, $row;
+    my $synonym = $row_data[2];
```

Comment:
Should we use the same variable name for different variable types (`$synonym`, `%synonym`)?

Reply:
Shouldn't use the same, it's updated now.